galaxy_ng/app/webserver_snippets/apache.conf

```diff
@@ -0,0 +1 @@
+Alias / /var/lib/pulp/assets/galaxy_ng/
```

Comment:
Just a placeholder, as it is not working and I haven't figured out how to do it on apache